swagger.json

```diff
@@ -33,21 +33,7 @@
           "200" : {
             "description" : "Success response",
             "schema" : {
-              "type" : "object",
-              "properties" : {
-                "id" : {
-                  "type" : "string",
-                  "example" : "oo000oo0001"
-                },
-                "state" : {
-                  "type" : "string",
-                  "example" : "in progress"
-                },
-                "request_id" : {
-                  "type" : "string",
-                  "example" : "0a72c00a-1332-4730-b19c-9131c3bcb57f"
-                }
-              }
+              "$ref" : "#/definitions/Resource"
```

Comment:
I don't agree that this is the correct response for a create. As documented on the whiteboard the response should be a RequestID, a ResourceID and a RequestStatus.   I don't think it makes sense to send them back the resource they just sent us.